### PR TITLE
#40 input widget iOS implementation

### PR DESCRIPTION
**Pull request by kovalandrew**

### CODE REVIEW

File: widgets/src/androidMain/kotlin/dev/icerock/moko/widgets/factory/SystemInputViewFactory.kt [hunk 1]
```diff
@@ -34,7 +34,8 @@ actual class SystemInputViewFactory actual constructor(
     private val textStyle: TextStyle?,
     private val labelTextColor: Color?,
     private val textHorizontalAlignment: TextHorizontalAlignment?,
-    private val textVerticalAlignment: TextVerticalAlignment?
+    private val textVerticalAlignment: TextVerticalAlignment?,
+    private val iosFieldBorderStyle: IOSFieldBorderStyle? // For ios only
```
Comment (Alex009) [inline widgets/src/androidMain/kotlin/dev/icerock/moko/widgets/factory/SystemInputViewFactory.kt]: it's should not be property. replace to 
`iosFieldBorderStyle: IOSFieldBorderStyle?`

File: widgets/src/commonMain/kotlin/dev/icerock/moko/widgets/style/view/IOSFieldBorderStyle.kt [hunk 2]
```diff
@@ -0,0 +1,8 @@
+package dev.icerock.moko.widgets.style.view
```
Comment (Alex009) [inline widgets/src/commonMain/kotlin/dev/icerock/moko/widgets/style/view/IOSFieldBorderStyle.kt]: copyright required here

File: widgets/src/iosMain/kotlin/dev/icerock/moko/widgets/factory/FloatingLabelInputViewFactory.kt [hunk 3]
```diff
@@ -30,25 +30,7 @@ import platform.QuartzCore.CAShapeLayer
...
-import platform.UIKit.addSubview
-import platform.UIKit.bottomAnchor
-import platform.UIKit.leadingAnchor
-import platform.UIKit.systemFontSize
-import platform.UIKit.systemGrayColor
-import platform.UIKit.systemRedColor
-import platform.UIKit.topAnchor
-import platform.UIKit.trailingAnchor
-import platform.UIKit.translatesAutoresizingMaskIntoConstraints
+import platform.UIKit.*
```
Comment (Alex009) [inline widgets/src/iosMain/kotlin/dev/icerock/moko/widgets/factory/FloatingLabelInputViewFactory.kt]: please disable star import and reformat again

File: widgets/src/iosMain/kotlin/dev/icerock/moko/widgets/factory/FloatingLabelInputViewFactory.kt [hunk 4]
```diff
@@ -102,6 +84,10 @@ actual class FloatingLabelInputViewFactory actual constructor(
             onFocusLost = {
                 widget.field.validate()
             }
+
+            if (textHorizontalAlignment != null) {
+                horizontalAlignment = horizontalAlignment
```
Comment (Alex009) [inline widgets/src/iosMain/kotlin/dev/icerock/moko/widgets/factory/FloatingLabelInputViewFactory.kt]: here bug

File: widgets/src/iosMain/kotlin/dev/icerock/moko/widgets/factory/SystemInputViewFactory.kt [hunk 5]
```diff
@@ -1,328 +1,97 @@
...
-import platform.UIKit.leadingAnchor
-import platform.UIKit.systemFontSize
-import platform.UIKit.systemGrayColor
-import platform.UIKit.systemRedColor
-import platform.UIKit.topAnchor
-import platform.UIKit.trailingAnchor
-import platform.UIKit.translatesAutoresizingMaskIntoConstraints
+import platform.Foundation.NSMutableAttributedString
+import platform.Foundation.create
+import platform.UIKit.*
```
Comment (Alex009) [inline widgets/src/iosMain/kotlin/dev/icerock/moko/widgets/factory/SystemInputViewFactory.kt]: reformat without star import

File: widgets/src/iosMain/kotlin/dev/icerock/moko/widgets/factory/FloatingLabelInputViewFactory.kt (hunk 4 above)
Comment (kovalandrew) [reply widgets/src/iosMain/kotlin/dev/icerock/moko/widgets/factory/FloatingLabelInputViewFactory.kt]: misstape fixed

File: widgets/src/commonMain/kotlin/dev/icerock/moko/widgets/style/view/IOSFieldBorderStyle.kt (hunk 2 above)
Comment (kovalandrew) [reply widgets/src/commonMain/kotlin/dev/icerock/moko/widgets/style/view/IOSFieldBorderStyle.kt]: added

File: widgets/src/iosMain/kotlin/dev/icerock/moko/widgets/factory/FloatingLabelInputViewFactory.kt (hunk 3 above)
Comment (kovalandrew) [reply widgets/src/iosMain/kotlin/dev/icerock/moko/widgets/factory/FloatingLabelInputViewFactory.kt]: reformatted

File: widgets/src/iosMain/kotlin/dev/icerock/moko/widgets/factory/SystemInputViewFactory.kt (hunk 5 above)
Comment (kovalandrew) [reply widgets/src/iosMain/kotlin/dev/icerock/moko/widgets/factory/SystemInputViewFactory.kt]: reformatted

**Pull request merged**